File: src/main/java/top/zbeboy/isy/service/graduate/design/GraduationDesignSubjectOriginTypeService.java
package top.zbeboy.isy.service.graduate.design;

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignSubjectOriginType;

import java.util.List;

/**
 * Created by zbeboy on 2017/6/13.
 */
public interface GraduationDesignSubjectOriginTypeService {

    /**
     * 查询全部
     *
     * @return 全部
     */
    List<GraduationDesignSubjectOriginType> findAll();
}

File: src/main/java/top/zbeboy/isy/service/graduate/design/GraduationDesignSubjectOriginTypeService.kt
package top.zbeboy.isy.service.graduate.design

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignSubjectOriginType

/**
 * Created by zbeboy 2018-01-26 .
 **/
interface GraduationDesignSubjectOriginTypeService {
    /**
     * 查询全部
     *
     * @return 全部
     */
    fun findAll(): List<GraduationDesignSubjectOriginType>
}